tutorials/math/tStudent.py

```diff
@@ -0,0 +1,80 @@
+## \file
+## \ingroup tutorial_math
+## \notebook
+## Example macro describing the student t distribution
+##
+## ~~~{.cpp}
+## root[0]: .x tStudent.C
+## ~~~
+##
+## It draws the pdf, the cdf and then 10 quantiles of the t Student distribution
+##
+## based on Magdalena Slawinska's tStudent.C
+##
+## \macro_image
+## \macro_code
+##
+## \author Juan Fernando Jaramillo Botero
+
+from ROOT import TH1D, TF1, TCanvas, kRed, kBlue
+import ROOT
+
+
+# this is the way to force load of MathMore in Cling
+ROOT.Math.MathMoreLibrary.Load()
+
+n = 100
+a = -5.
+b = 5.
+pdf = TF1("pdf", "ROOT::Math::tdistribution_pdf(x,3.0)", a, b)
+cum = TF1("cum", "ROOT::Math::tdistribution_cdf(x,3.0)", a, b)
+
+quant = TH1D("quant", "", 9, 0, 0.9)
+
+for i in range(1, 10):
+    quant.Fill((i-0.5)/10.0, ROOT.Math.tdistribution_quantile((1.0 * i) / 10,
+                                                              3.0))
+
+xx = []
+xx.append(-1.5)
+for i in range(1, 9):
+    xx.append(quant.GetBinContent(i))
```

Comment:
Something to make this nice translation even more Pythonic: why not list comprehensions?